### ISSUE TITLE
The option 'family' for tex2svg / tex2chtml has no default value

### ISSUE THREAD

**Issue by christianp** [mathjax/MathJax#2512]
I use `MathJax.getMetricsFor` to get an options object for `tex2chtml` or `tex2svg`, [as described in the typesetting page in the docs](http://docs.mathjax.org/en/latest/web/typeset.html#obtaining-the-output-metrics).

It includes an attribute `family` whose value is the empty string. When I pass the object to `tex2chtml`, I get an error "Invalid option "family" (no default exists)". So it looks easy to fix, but I don't know where to look for the default values.

Here's a node script which produces the error:

```
const MathJax = require('mathjax');

MathJax.init({
    loader: {load: ['input/tex', 'output/svg']}
}).then(MathJax => {
    const svg = MathJax.tex2svg('2x',{family:''});
}).catch(err => {
    console.error(err);
});
```

(I'd love to fix this myself, but I can't remember how to run unminified MathJax v3)

**Comment by dpvc** [comment]
Ok, this is caused by the addition of the `family` property to the extended metric data used by the HTML MathDocument class, which was not taken into account by the `convert()` macro that underlies `tex2svg()` and the other similar conversion functions.  Unfortunately, the default list is not in a place that is easily adjusted on the fly.  Here is a configuration that can be used to replace the `convert()` function with a fixed one.  This not only allows the `family` property, but actually handles it to provide support for `mtextInheritFont` and the other similar output options.

``` html
<script>
MathJax = {
  startup: {
    ready() {
      const {HTMLDocument} = MathJax._.handlers.html.HTMLDocument;
      const {userOptions} = MathJax._.util.Options;
      const {STATE} = MathJax._.core.MathItem;
      HTMLDocument.prototype.convert = function (math, options = {}) {
        let {family, format, display, end, ex, em, containerWidth, lineWidth, scale} = userOptions({
          family: '', format: this.inputJax[0].name, display: true, end: STATE.LAST,
          em: 16, ex: 8, containerWidth: null, lineWidth: 1000000, scale: 1
        }, options);
        if (containerWidth === null) {
          containerWidth = 80 * ex;
        }
        const jax = this.inputJax.reduce((jax, ijax) => (ijax.name === format ? ijax : jax), null);
        const mitem = new this.options.MathItem(math, jax, display);
        if (this.outputJax.options.mtextInheritFont) {
          mitem.outputData.mtextFamily = family;
        }
        if (this.outputJax.options.merrorInheritFont) {
          mitem.outputData.merrorFamily = family;
        }
        mitem.start.node = this.adaptor.body(this.document);
        mitem.setMetrics(em, ex, containerWidth, lineWidth, scale);
        mitem.convert(this, end);
        return (mitem.typesetRoot || mitem.root);
      };
      MathJax.startup.defaultReady();
    }
  }
};
</script>
```

It is longer than I would like, but you have to replace the entire `convert()` function in order to make the needed changes.  I will make a PR that fixes the function for the next release.  We'll probably have to make a point release to fix this.  Sigh.

**Comment by aaryanUV** [comment]
When will this fix be released?

**Comment by dpvc** [comment]
We are hoping to get a release out later this week.
